### PR TITLE
Added ORT in observatories.json

### PR DESCRIPTION
**Pull request by abhisrkckl**

### CODE REVIEW

File: src/pint/data/runtime/observatories.json [hunk 1]
```diff
@@ -255,6 +255,14 @@
         ],
         "origin": "The Giant Metrewave Radio Telescope.\nThe origin of this data is unknown but as of 2021 June 8 it agrees exactly with\nthe values used by TEMPO and TEMPO2.\n"
     },
+    "ort": {
+        "itrf_xyz": [
+            1442712.95,
+            6087044.73,
+            1251052.35
+        ],
+        "origin": "The Ooty Radio Telescope.\nCoordinates provided by Jaikhomba Singha.\n"
```
Comment (dlakaplan) [inline src/pint/data/runtime/observatories.json:264]: Maybe just specify the date here in case there are questions in the future?

**Comment by dlakaplan** [comment]
ORT doesn't require any special clock files?

**Comment by abhisrkckl** [comment]
Not as far as I know. I'll double check.

**Comment by abhisrkckl** [comment]
I have checked that ORT does not have any clock files. Its TOAs are measured in UTC(gps).

**Comment by abhisrkckl** [comment]
I'll ask

File: src/pint/data/runtime/observatories.json [hunk 2]
```diff
@@ -255,6 +255,14 @@
         ],
         "origin": "The Giant Metrewave Radio Telescope.\nThe origin of this data is unknown but as of 2021 June 8 it agrees exactly with\nthe values used by TEMPO and TEMPO2.\n"
     },
+    "ort": {
+        "itrf_xyz": [
+            1442712.95,
+            6087044.73,
+            1251052.35
+        ],
+        "origin": "The Ooty Radio Telescope.\nCoordinates provided by Jaikhomba Singha on 25 Jan 2023.\n"
```
Comment (aarchiba) [inline src/pint/data/runtime/observatories.json:264]: Maybe a note in the description explaining why it doesn't need clock files?

Comment (abhisrkckl) [reply src/pint/data/runtime/observatories.json]: Added.

**Comment by abhisrkckl** [comment]
I have tested this with some real data and it works.

**Pull request merged**